### PR TITLE
Accept mixed aware/naive start/end dates

### PR DESCRIPTION
**Pull request by hobarrera**
See #260

### CODE REVIEW

**Comment by untitaker** [comment]
Could you add a testcase for the other side as well? Should be not too verbose if you use pytest.parametrize to generate DTSTART/DTEND lines, and insert them into a template.

**Comment by hobarrera** [comment]
Oh, I'd already added the opposite case into the same file (since otherwise I'm dropping coverage percentage).

File: tests/controller_test.py [hunk 1]
```diff
@@ -76,3 +76,24 @@ def test_import(self, coll_vdirs):
...
+        )
+        start_date = aux.BERLIN.localize(datetime.datetime(2015, 5, 29))
+        end_date = aux.BERLIN.localize(datetime.datetime(2015, 5, 31))
+        events = list(coll.get_localized(start_date, end_date))
+        assert len(events) == 2
+        events = sorted(events)
+        assert events[1].start_local == \
+            aux.BERLIN.localize(datetime.datetime(2015, 5, 30, 12, 0))
+        assert events[1].end_local == \
+            aux.BERLIN.localize(datetime.datetime(2015, 5, 30, 16, 0))
```
Comment (untitaker) [inline tests/controller_test.py]: Could you add some assertions here? This only checks the results of one case (though coverage is full)

Comment (hobarrera) [inline tests/controller_test.py]: Oops!

**Comment by hobarrera** [comment]
Okay to merge?

**Comment by untitaker** [comment]
IMO :+1:

**Comment by geier** [comment]
Go ahead, please!

Quoting Markus Unterwaditzer (2016-02-26 14:43:07)

**Comment by untitaker** [comment]
@hobarrera You'll need to rebase :(

**Pull request merged**